pyproject.toml

```diff
@@ -58,6 +58,7 @@ dev = [
     "urllib3<2.0.0",
     "wandb",
     "expecttest",
+    "fvcore",  # For MFU calculation
```

Comment:
n00b question: is MFU calculation simple enough that it could be an utility, instead of a new dependency?

Reply:
Good Q! Let me poke around OpenHands and see if we can have a solution without additional dependency

Reply:
torchtitan also computes MFU. They are a great reference implementation. Maybe it is worth taking a look at how they do it. 

https://github.com/pytorch/torchtitan/blob/9a4eebe1c9a692027f0c8ce88f7caf89af3dd616/train.py#L70

Reply:
edit: I just took a look at it and I dont like it too much! haha. Maybe there are other alternatives. If nothing simple, it might be ok to have a dependency, if its not a heavy one.

Reply:
Thanks for the pointer!

I actually found one from megatron-LM: https://github.com/NVIDIA/Megatron-LM/blob/main/megatron/training/one_logger_utils.py

Just send off to OpenHands and see how it looks like at the end

<img width="386" alt="image" src="https://github.com/user-attachments/assets/8b165d4c-0025-4d48-8055-90579552266d">